src/tags/visual/View.js

```diff
@@ -110,8 +112,15 @@ const HtxView = observer(({ item }) => {
     style["display"] = "none";
   }
 
+  const className = [
+    item.className,
+    styles.viewContainer,
```

Comment:
since its already setup to do the conditional className building, why not just do this instead:

```tsx
isFF(FF_DEV_2395) && styles.viewContainer
```

Reply:
done